src/components/Post/style.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/constants';
+
+.post {
+  padding-left: 10px;
+  margin-bottom: 15px;
+  font-family: $roboto-font;
+  border-bottom: 1px solid $gray;
+
+  &__header {
+    display: flex;
+
```

Comment:
Oops. When you have an object we do not leave an extra line inside the object in the last line